packages/builders/src/util/slashCommandUtil.ts

```diff
@@ -0,0 +1,9 @@
+import type { LocaleString } from 'discord-api-types/v10';
+
+export function flattenLocaleMap(
+	localeMap: Map<LocaleString, string> | Partial<Record<LocaleString, string>>,
+): [LocaleString, string][] {
+	return localeMap instanceof Map
+		? [...localeMap.entries()]
+		: ([...Object.entries(localeMap)] as [LocaleString, string][]);
```

Comment:
Isn't spreading the array redundant here?

Reply:
No, because entries isn't an array, it's an iterable

Reply:
Sure? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

Reply:
Oops, I stand corrected

Reply:
Take a look again @suneettipirneni